scout/server/blueprints/variant/templates/variant/sv-variant.html

```diff
@@ -244,6 +244,13 @@
           <span class="d-inline">Type</span>
           <span class="d-inline float-end">{{ variant.sub_category|upper }}</span>
         </li>
+
+        {% if variant.genes and variant.genes|length <= 5 %}
```

Comment:
This is super arbitrary. Could be variant.genes and variant.length < something as well?

Reply:
Indeed, or some general combination of them. As long as it's a relatively small thing, the transcripts tab below will also show stuff, so let's perhaps just leave it at this for now.